app/views/shared/views/_show_containers_dashboard.html.haml

```diff
@@ -0,0 +1,125 @@
+.container-fluid.container-tiles-pf.containers-dashboard.miq-dashboard-view{"ng-app"        => "containerDashboard",
```

Comment:
Oh, that's right.. can you (or @abonas) reply to Eric there? I'm *almost* sure we do want font icons but maybe I'm wrong?